Editor/ExportUtility.cs

```diff
@@ -1133,36 +1184,41 @@ public static Mesh ExportQuad(string meshName, float width, float height)
         /// <summary>
         /// returns texture2d baked from canvas target
         /// </summary>
-        public static Texture2D CanvasTextureBake(Transform target, int resolution = 512)
+        private static Texture2D TextureBake(Transform target, ExportQuadType type, float width, float height, int resolution = 1024)
         {
-            GameObject cameraGo = new GameObject("Temp_Camera");
+            GameObject cameraGo = new GameObject("Temp_Camera " + target.gameObject.name);
             Camera cam = cameraGo.AddComponent<Camera>();
 
             //snap camera to canvas position
             cameraGo.transform.rotation = target.rotation;
             cameraGo.transform.position = target.position - target.forward * 0.05f;
 
-            var width = target.GetComponent<RectTransform>().sizeDelta.x * target.localScale.x;
-            var height = target.GetComponent<RectTransform>().sizeDelta.y * target.localScale.y;
-            if (Mathf.Approximately(width, height))
+            if (type == ExportQuadType.Canvas) // use rect bounds for canvas
```

Comment:
I wish I could leave out the condtional branches for `ExportQuadType` here - but it seems necessary. To fix it will be a larger refactor, out of scope for this ticket. Potential to look at later for larger scale refactor?